docs/man/clamd.conf.5.in

```diff
@@ -83,7 +83,7 @@ Log additional information about the infected file, such as its size and hash, t
 Default: no
 .TP
 \fBPidFile STRING\fR
```

Comment:
STRING -> FILE (referenced in next line)

Or: "Write daemon's pid to filename contained in STRING"

Reply:
Hm same issue exists in `clamav-milter.conf.5.in`.  Will fix both.   

It's a pretty deep rabbithole to make it use `FILE` for both, or `STRING` for both.  So many other options just say `STRING` and then something like "the specified file."

So I think I'll change it to "the specified file." to match those.  Good catch!